easybuild/easyconfigs/n/NVHPC/NVHPC-20.7-GCC-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+name = 'NVHPC'
+version = '20.7'
+local_gccver = '9.3.0'
+versionsuffix = '-GCC-%s' % local_gccver
+
+homepage = 'https://developer.nvidia.com/hpc-sdk/'
+description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (previously: PGI)"""
+
+toolchain = SYSTEM
+
+sources = ['nvhpc_2020_%(version_major)s%(version_minor)s_Linux_x86_64_cuda_multi.tar.gz']
+
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.34', '', ('GCCcore', local_gccver)),
```

Comment:
Why not simply make `GCC/9.3.0` the `toolchain`?

Reply:
I took that from the old`PGI`. @damianam, do you have anything to add?

Reply:
`NVHPC` is a compiler, the same as `iccifort` so this seems appropriate to me ;-)